### PR TITLE
Add a gprc template

### PR DESCRIPTION
**Pull request by asbfall**
- [X] Add a grpc tempate that instanciate a server that users can ping
- [X] Specific documentation about how to build and start a grpc server in python
- [X] Dockerfile for a containerized application
- [x] Server unit test file
- [x] Unit tests

### CODE REVIEW

File: src/templates/gRPC-api/src/server/server.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022, Ansys Inc. Unauthorised use, distribution or duplication is prohibited
```
Comment (jgd10) [inline src/templates/gRPC-api/src/server/server.py:1]: Is this license in conflict with the MIT license on the project?

**Review by jgd10 — APPROVED** [review]
Looks great! I have a couple of comments as well :)

File: src/templates/gRPC-api/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```shell
+ python -m grpc_tools.protoc -I ../../protobufs --python_out=../stubs --grpc_python_out=../stubs ../../protobufs/*.proto
+```
+Here’s a breakdown: 
+- **python -m grpc_tools.protoc** runs the protobuf compiler, which will generate Python code from the protobuf code.
+
+- **-I ../../protobufs** tells the compiler where to find files that your protobuf code imports. You don’t actually use the import feature, but the -I flag is required nonetheless.
+
+- **--python_out=../stubs --grpc_python_out=../stubs** tells the compiler where to output the Python files. In this template we generate all the compiled code under the a `stubs` to group tham and to use the same for the client and the server.
```
[the remark this answers is not in the document]
Comment (asbfall) [reply src/templates/gRPC-api/README.md:30]: yes two typos !

File: src/templates/gRPC-api/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+### Deployment
+
+Docker is perfect for deploying a Python microservice because you can package all the dependencies and run 
+the microservice in an isolated environment
+
+##### Using docker CLI
+
+````shell
+docker run -p 127.0.0.1:50056:50051/tcp <your-tag>
```
Comment (jacobevansgit) [inline src/templates/gRPC-api/README.md:68]: @asbfall , 

Note: with docker run you expose it on port 50056 and with docker-compose up you use 50051.

Comment (asbfall) [reply src/templates/gRPC-api/README.md:68]: you are correct. I'll change the port binding here

**Pull request merged**